pkg/bolttools/utils.go

```diff
@@ -25,7 +25,7 @@ import (
 func get(bucket *bolt.Bucket, key []byte) ([]byte, error) {
 	value := bucket.Get(key)
 	if value == nil {
-		return nil, fmt.Errorf("Key '%s' doesn't exist in the bucket: %#v", key, bucket)
+		return nil, fmt.Errorf("Key '%s' doesn't exist in the bucket", key)
```

Comment:
I don't think `bolt.Bucket` contains any human-readable info that's valuable for virtlet debugging, or maybe I'm wrong here?